workitem_whitebox_test.go

```diff
@@ -20,15 +22,25 @@ var DB *gorm.DB
 var rwiScheduler *remoteworkitem.Scheduler
 
 func TestMain(m *testing.M) {
+	var err error
+
+	if err = configuration.SetupConfiguration(configuration.DefaultConfigFilePath); err != nil {
+		panic(fmt.Errorf("Failed to setup the configuration: %s", err.Error()))
+	}
+
 	if _, c := os.LookupEnv(resource.Database); c != false {
```

Comment:
The env var lookup for resource.Database is a copy of the code in resource.Require(). Let's not copy if not necessary.